vermouth/data/mappings/elnedyn/modifications.elnedyn21.mapping

```diff
@@ -2,7 +2,7 @@
 [ from ]
 universal
 [ to ]
-elnedyn22p
+elnedyn21
```

Comment:
Is this intended?

Reply:
nope will fix it

Reply:
this came from cleaning the elnedyn directory. I guess there should be a modifications file for each version because modifications cannot stack force-fields but mapping files can ?! To be honest the mapping directories are a mess